test/integration/targets/iptables/tasks/chain_management.yml

```
@@ -45,6 +45,26 @@
       - result is not failed
       - '"FOOBAR-CHAIN" in result.stdout'
 
+- name: add rule to foobar chain
+  become: true
+  iptables:
+    chain: FOOBAR-CHAIN
+    source: 0.0.0.0
+    destination: 0.0.0.0
+    jump: DROP
+    comment: "FOOBAR-CHAIN RULE"
+
+- name: get the state of the iptable rules after rule is added to foobar chain
+  become: true
+  shell: "/usr/sbin/iptables -L"
```

Thanks for adding back a test!

```suggestion
  shell: "{{ iptables_bin }} -L"
```